packages/fast-components-react-msft/app/utilities/additional-props.plugin.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import {
+    ChildOptionItem,
+    Plugin,
+    PluginProps,
+} from "@microsoft/fast-data-utilities-react";
+
+export default class MapChildrenPropToCallbackPassingClassName extends Plugin<
+    PluginProps
+> {
+    public resolver(data: any, childOption?: ChildOptionItem): any {
+        return (
+            className: string,
+            disabled?: boolean,
+            appearance?: string
+        ): React.ReactNode => {
+            const callbackArguments: any = {
+                className,
+            };
+            callbackArguments.disabled = disabled !== undefined ? disabled : undefined;
+            callbackArguments.appearance =
+                appearance !== undefined ? appearance : undefined;
```

Comment:
Same here - if `appearance` needs to be a string, why not check for `typeof string`?